web/backend/test/routes/root.test.ts

```diff
@@ -98,6 +99,9 @@ test('runtime is running', async (t) => {
     },
   })
 
+  // Keep the Open API definition of the backend service up-to-date
+  await fs.writeFile('openapi.json', JSON.stringify(json))
```

Comment:
I think this is problematic. You need a different step to generate this.

Reply:
Good point; it was somewhat of a hack, but in fact, I wasn't very happy about it. The current solution is more reliable...